### PR TITLE
Website v1.0

### PR DESCRIPTION
**Pull request by MidouWebDev**
## Description

Changed the repo structure
Added the Invite Bot Link
Fixed SEO issues

### CODE REVIEW

**Review by j-dogcoder — APPROVED** [review]

**Comment by j-dogcoder** [comment]
@MidouWebDev Please resolve conflicts!

**Comment by MidouWebDev** [comment]
@j-dogcoder done !

File: 404.html [hunk 1]
```diff
@@ -1,62 +1,68 @@
...
+<body>
+  <div class="bgimg">
+    <div class="middle">
+      <h1>404</h1>
+      <hr>
+      <p>You have reached a page that doesn't exist or is having issues. If you think this is a mistake, please report this in the <a href="https://discord.gg/HSupF99kpq">official discord server.</a></p>
+    </div>
+  </div>
+</body>
+</html>
```
Comment (Panquesito7) [inline 404.html:68]: Missing an endline.
```suggestion
</html>

```

Comment (MidouWebDev) [reply 404.html:68]: I am working on a new 404 page

**Pull request merged**